components/automate-ui/src/app/modules/policy/add-members/policy-add-members.component.ts

```diff
@@ -156,28 +155,9 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
     this.store.select(addPolicyMembersStatus).pipe(
       takeUntil(this.isDestroyed),
       filter(addState => this.addingMembers && !pending(addState)))
```

Comment:
This gets greatly simplified since now we want success or failure to do the same thing--close the full page dialog.
(For this page, there are no field-specific errors to render with the page open, as mentioned in the preamble.)